sru-scripts/manual/gce-sru

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# Manual GCE upgrade and clean install validation Xenial, Bionic, Cosmic
+set -ex
+SRU_VARS=./sru-scripts/sru-vars.template
+
+if [ ! -f $SRU_VARS ]; then
+   echo "Missing config environment file $SRU_VARS."
+   exit 1
+fi
+
+. $SRU_VARS
+
+parse_args $0 $@
+assert_expected_vars PROPOSED_SCRIPT SAMPLE_CLOUDCONFIG SSHOPTS GCE_ZONE \
+    NETCFG_FILE
+create_samplecloudconfig SAMPLE_CLOUDCONFIG
+create_setup_proposed_script
+ 
+echo "### BEGIN $UBUNTU_RELEASE"
+
+case $UBUNTU_RELEASE in
+    xenial) family=ubuntu-1604-lts;;
+    bionic) family=ubuntu-1804-lts;;
+    cosmic) family=ubuntu-1810;;
+    disco) family=ubuntu-1904;;
+    eoan) family=ubuntu-1910;;
+    focal) family=ubuntu-2004;;
+    *) echo "!! UPDATE FAMILY CASE STATEMENT !!"; exit 1;;
+esac
+
+gcloud compute instances create $UBUNTU_RELEASE-sru-test --image-family $family --image-project ubuntu-os-cloud-devel --metadata-from-file user-data=${SAMPLE_CLOUDCONFIg} --zone=$GCE_ZONE;
```

Comment:
SAMPLE_CLOUDCONFIg -> SAMPLE_CLOUDCONFIG

Reply:
Thanks for the review @TheRealFalcon fixed and pushed both.